### PR TITLE
Added 'add an item of spend' functionality

### PR DESCRIPTION
**Pull request by stuartmccoll**
## Proposed changes

This pull request adds 'add an item of spend' functionality.

## Further comments

N/A.

## Screenshot (if applicable)

N/A.

## Checklist

- [ ] Added tests for new functionality, or updated existing tests
- [ ] Updated `README.md` if necessary

### CODE REVIEW

File: app/views/pre_application/expression_of_interest/heritage_focus/show.html.erb [hunk 1]
```diff
@@ -2,7 +2,7 @@
   render partial: "partials/page_title",
          locals: {
              model_object: @pre_application.pa_expression_of_interest,
-      page_title: t('pa_expression_of_interest.heritage_focus.page_title')
+      page_title: t('pa_expression_of_intereest.heritage_focus.page_title')
```
Comment (ptrelease) [inline app/views/pre_application/expression_of_interest/heritage_focus/show.html.erb:5]: Correct this

File: app/views/funding_application/payment_requests/add_item_of_spend/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,430 @@
...
+              } if @spend.errors[:description].any?
+      %>
+
+      <%=
+        f.text_area :description,
+                    rows: 10,
+                    class: "govuk-textarea govuk-js-character-count" \
+                            "#{' govuk-textarea--error' if
+                        @spend.errors[:description].any?}",
+                    "aria-describedby" => "description-hint spend_description-info"
```
Comment (stuartmccoll) [inline app/views/funding_application/payment_requests/add_item_of_spend/show.html.erb:108]:
```suggestion
                    "aria-describedby" => "spend_description-info"
```

File: app/views/funding_application/payment_requests/add_item_of_spend/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,430 @@
+<%=
+  render partial: "partials/page_title",
+         locals: {
+             model_object: @payment_request,
```
Comment (stuartmccoll) [inline app/views/funding_application/payment_requests/add_item_of_spend/show.html.erb:4]:
```suggestion
             model_object: @spend,
```

File: db/migrate/20210209131311_add_static_cost_types.rb [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    execute "insert into cost_types (name, created_at, updated_at) values ('Expenses for staff', now(), now());"
+    execute "insert into cost_types (name, created_at, updated_at) values ('Expenses for volunteers', now(), now());"
+    execute "insert into cost_types (name, created_at, updated_at) values ('Other', now(), now());"
+    execute "insert into cost_types (name, created_at, updated_at) values ('Publicity and promotion', now(), now());"
+    execute "insert into cost_types (name, created_at, updated_at) values ('Evaluation', now(), now());"
+    execute "insert into cost_types (name, created_at, updated_at) values ('Contingency', now(), now());"
+  end
+
+  def down
+    execute "delete from cost_types where name = 'New staff'";
```
Comment (stuartmccoll) [inline db/migrate/20210209131311_add_static_cost_types.rb:24]: The semicolons need to go inside the `"`'s.

**Review by ptrelease — APPROVED** [review]

**Pull request merged**